spec/rubocop/cli_spec.rb

```diff
@@ -1627,6 +1627,23 @@ def abs(path)
       end
     end
 
+    describe '-e/--extra-details' do
+      it 'shows extra details' do
```

Comment:
It won't be too different from the one you've got. Add
```ruby
        create_file('.rubocop.yml',
                    ['TrailingWhitespace:',
                     '  Details: Trailing space is just sloppy.'])
```
right here :point_down: